bin/citrea/provers/risc0/light-client-prover-mock/src/bin/light_client_prover_mock.rs

```diff
@@ -32,6 +32,9 @@ pub fn main() {
     let start_cycles = env::cycle_count();
 
     let input: LightClientCircuitInput<MockDaSpec> = guest.read_from_host();
+    let batch_prover_journal = input.batch_prover_journal.clone();
+    let batch_prover_method_id = input.batch_prover_method_id.clone();
+    Risc0Guest::verify(&batch_prover_journal.unwrap(), &batch_prover_method_id).unwrap();
 
     let da_verifier = MockDaVerifier {};
 
```

Comment:
instead of doing verification here, let's pass guest var to the run_circuit function, and verify stuff there when we want